pyart/io/output_to_geotiff.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import print_function, division
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import os
+try:
+    from osgeo import gdal
+    IMPORT_FLAG = True
+except ImportError:
+    IMPORT_FLAG = False
+
+
+def write_grid_geotiff(filename, grid, field=None, rgb=False, level=None,
```

Comment:
Made `field` a required argument. Also reordered the requirement arguments to make them more logical in order.